modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2539,6 +2539,7 @@ def concat(
             New Modin DataFrame.
         """
         axis = Axis(axis)
+        new_widths = None
```

Comment:
For `new_lengths` isn't possible do the same because due to usage of `rebalance_partitions` inside `partition_manager.concat`.

Reply:
Does the rebalance only affect the lengths?

Reply:
Yes, https://github.com/modin-project/modin/blob/88c3c33cf9b8aab5f5b7077555bea9f2d8466534/modin/core/dataframe/pandas/partitioning/partition_manager.py#L623